cmd/api/src/services/storage/fileserviceresolver.go

```
@@ -192,6 +193,12 @@ func resolveFileServiceDefinitions(cfg config.Configuration, definitions []FileS
 		definitionsByName[definition.Name] = struct{}{}
 
 		if serviceConfiguration, configured = cfg.Storage.FileServices[string(definition.Name)]; configured {
+			slog.Info(fmt.Sprintf(
```

golangci-lint (2.12.2)

[major] message should be a string literal or a constant

(sloglint)